test/unit.test.js

```diff
@@ -218,6 +218,21 @@ for (const { testName, isRoot } of unitTests) {
         expect(foundMatchingBinary).toBe(true)
       }
 
+      if (testName === 'zeromq-node-gyp') {
```

Comment:
Maybe we should move this into line 206 so the logic is the same for both `microtime-node-gyp` and `zeromq-node-gyp`

Reply:
Done.